S21/ajayrazz99/wordChanger.py

```diff
@@ -0,0 +1,48 @@
+def LowerCaseConverter(word,lang):
+    lang=lang
+    word=word
+    if(lang=='turkey'or lang=='az' or lang=='tr'):      #converts uppercase to lowercase
```

Comment:
This can be optimised to if(lang=='turkey' or 'az' or 'tr'):

Reply:
Yes, thank you. I'll make sure to change that.